step-plans/step-plans-base-artefacts/src/test/java/step/reporting/ResolvedPlanBuilderTest.java

```diff
@@ -39,37 +40,47 @@ public void after() {
     public void get() throws IOException, InterruptedException {
         Plan plan = PlanBuilder.create()
                 .startBlock(BaseArtefacts.for_(1, 10))
-                .add(BaseArtefacts.echo("'test'"))
+                .add(echo("'test'"))
                 .startBlock(BaseArtefacts.for_(1, 5))
-                .add(BaseArtefacts.echo("'Echo 2'"))
-                .add(BaseArtefacts.echo("'Echo 3'"))
+                .add(echo("'Echo 2'"))
+                .add(echo("'Echo 3'"))
                 .endBlock()
                 .endBlock().build();
         PlanRunnerResult result = engine.execute(plan);
         result.printTree();
 
         // Sleep a few ms to ensure that the report node timeseries is flushed
-        Thread.sleep(100);
+        //Thread.sleep(500);
 
         AggregatedReportViewBuilder aggregatedReportViewBuilder = new AggregatedReportViewBuilder(engine.getExecutionEngineContext(), result.getExecutionId());
         AggregatedReportView node = aggregatedReportViewBuilder.buildAggregatedReportView();
+        System.out.println("----------------------");
```

Comment:
Not sure if we should replace this by proper logging even though it's just a test